rucio-dataset-mon/src/go/README.md

```diff
@@ -0,0 +1,9 @@
+
+Ref: https://dev.to/hackmamba/build-a-rest-api-with-golang-and-mongodb-gin-gonic-version-269m
+
+How to run:
+
+There is a running MongoDB instance in personal K8s cluster that is used for test in this code.
+
+Just run Go: `go run main.go`
```

Comment:
Usually, `go run` should be avoided since it does not have all dependencies. Instead, please generate within source code the `go.mod` and `go.sum` files and then use go build to create final executable. To create them you should do:
```
# cd to your repo area
# run go mod init with your repo or any other name
go mod init github.com/<you repo>
# get all dependencies for your code
go mod tidy
```